### PR TITLE
Fix sorting label

### PR DESCRIPTION
**Pull request by andyw8**
We changed the behaviour of this but didn't update the link text.

### CODE REVIEW

File: app/views/books/index.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 <% if books.any? %>
   <%= link_to("Sort by Title", sort_path("title")) %> |
-  <%= link_to("Sort by Date Added", sort_path(Book::SORT_CRITERIA_ADDED)) %>
+  <%= link_to("Sort by Newest First", sort_path(Book::SORT_CRITERIA_ADDED)) %>
```
Comment (taylorrf) [inline app/views/books/index.html.erb]: We can update here to send the link text `"Sort by Newest First"` to use I18n

File: app/views/books/index.html.erb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 <% if books.any? %>
   <%= link_to("Sort by Title", sort_path("title")) %> |
```
Comment (taylorrf) [inline app/views/books/index.html.erb]: Same here..

File: app/views/books/index.html.erb (hunk 1 above)
Comment (andyw8) [inline app/views/books/index.html.erb]: Good point

Comment (andyw8) [inline app/views/books/index.html.erb]: I've also renamed the constant.

**Comment by taylorrf** [comment]
:+1:

**Pull request merged**